client/app/src/main/java/com/microsoft/research/karya/data/model/karya/FileCreator.kt

```diff
@@ -7,8 +7,8 @@
  */
 package com.microsoft.research.karya.data.model.karya
 
+import com.google.gson.annotations.SerializedName
+
 enum class FileCreator {
-  karya_server,
-  karya_box,
-  karya_client
+  @SerializedName("worker") WORKER
```

Comment:
Why do we even need this if the creator will always be a `WORKER`

Reply:
I forgot to add the value for server, in case of input files (which is not right now)